Sample-App/app/src/main/java/com/adobe/marketing/mobile/sampleapp/EdgeIdentityTab.java

```diff
@@ -105,6 +117,80 @@ public void call(String ecid) {
+                getAdvertisingIdClientInfo(new AdobeCallback<AdvertisingIdClient.Info>() {
+                    @Override
+                    public void call(AdvertisingIdClient.Info info) {
+                        Handler handler = new Handler(Looper.getMainLooper());
+
+                        String adIdText = "Unable to get valid AdvertisingIdClient.Info";
+                        String trackingAuthorizationText = "See console for error logs";
+
+                        if (info != null) {
+                            if (info.isLimitAdTrackingEnabled()) {
+                                adIdText = "";
+                                trackingAuthorizationText = "Ad tracking disabled";
+                            } else {
+                                adIdText = info.getId();
+                                trackingAuthorizationText = "Ad tracking enabled";
+                            }
+                            // Update ad ID through MobileCore only when a valid Info instance is available
+                            // to fetch the latest state
+                            MobileCore.setAdvertisingIdentifier(adIdText);
+                        }
+
+                        String finalAdId = adIdText;
+                        String finalTrackingAuthorizationText = trackingAuthorizationText;
+                        handler.post(() -> {
+                            TextView gaidTextView = getView().findViewById(R.id.label_edge_identity_gaid_placeholder);
+                            gaidTextView.setText(finalAdId);
+                            TextView adTrackingTextView = getView().findViewById(R.id.label_edge_identity_ad_tracking_enabled);
+                            adTrackingTextView.setText(finalTrackingAuthorizationText);
+                        });
+                    }
+                });
+            }
+        });
+
+        btnSetAdId.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(final View v) {
+                EditText adIdText = getView().findViewById(R.id.text_ad_id);
+                String adId = adIdText.getText().toString();
+                MobileCore.setAdvertisingIdentifier(adId);
+            }
+        });
+
+        btnSetAdIdNull.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(final View v) {
+                MobileCore.setAdvertisingIdentifier(null);
+            }
+        });
+
+        btnSetAdIdAllZeros.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(final View v) {
+                MobileCore.setAdvertisingIdentifier(ZERO_ADVERTISING_ID);
+            }
+        });
+
+        btnSetAdIdEmptyString.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(final View v) {
+                MobileCore.setAdvertisingIdentifier("");
+            }
+        });
```

Comment:
As this is the public Sample App, we should provide examples of actual API usage (i.e. your update Ad ID button) and not include debugging code (i.e. the 4 "set" buttons). I see you've updated the Edge Identity test app with UI buttons for passing various Ad ID values, and that's great and the proper place for them, but I don't feel the need to include them in this app.